include/pm/device.h

```diff
@@ -90,12 +93,11 @@ typedef int (*pm_device_action_cb_t)(const struct device *dev,
  * @brief Device PM info
  */
 struct pm_device {
-#ifdef CONFIG_PM_DEVICE_RUNTIME
+#if defined(CONFIG_PM_DEVICE_RUNTIME) || defined(__DOXYGEN__)
```

Comment:
why this conditional ? `defined(__DOXYGEN__)`

Reply:
It can be used to unconditionally include sections of the docs without the need of adding predefines in the Doxygen side. A live example:

https://github.com/zephyrproject-rtos/zephyr/blob/811a09bd83052a40220d70eb3550048915d0be23/include/drivers/pinctrl.h#L58-L65

It allows for more fine-grained inclusion compared to using predefines.